EPFIngester.py

```diff
@@ -537,22 +537,14 @@ def psycopg2_async_set_client_encoding(conn, encoding):
                 )
 
         if self.isPostgresql:
-            wait(conns[0])
-            conns[0].close()
-            wait(conns[1])
-            conns[1].close()
-            wait(conns[2])
-            conns[2].close()
-            wait(conns[3])
-            conns[3].close()
-            wait(conns[4])
-            conns[4].close()
-            wait(conns[5])
-            conns[5].close()
-            wait(conns[6])
-            conns[6].close()
-            wait(conns[7])
-            conns[7].close()
+            for i in range(0, 7):
```

Comment:
Would `for conn in conns` be better?
Then there's no need to encode the size of conns.
I assume that's all of them? I kinda expected there to be 8 for some reason so it should otherwise be `range(8)`.

Reply:
Yes, that would be a more sensible way to do it, wouldn't it.

Reply:
cf07400